### PR TITLE
Move items

### PR DESCRIPTION
**Pull request by richardiphillips**
Palletise cartons and items.
Cartonise items. 
Recalculate functions. 

Manually type consignment number in wand for shug.

### CODE REVIEW

File: src/Service.Host/client/src/components/consignments/ItemsTab.js [hunk 1]
```diff
@@ -87,6 +101,206 @@ function ItemsTab({
...
+        let weight = 0;
+        const setValues = item => {
+            if (item.itemType === 'C'){
+                weight += item.itemBaseWeight;
+            } else {
+                weight += item.weight;
+            }
+        };
+
+        // eslint-disable-next-line prettier/prettier
```
Comment (iaincrawf) [inline src/Service.Host/client/src/components/consignments/ItemsTab.js:248]: what doesn't it like? I've never actually used the disable-next-line thing before

**Review by iaincrawf — APPROVED** [review]
Looks good from my limited consignment-y knowledge. Also something about manually checking key-press codes that I love for some reason

File: src/Service.Host/config-example.env [hunk 2]
```diff
@@ -6,11 +6,11 @@ RABBIT_SERVER=localhost
...
-APP_ROOT=http://localhost:61798
+APP_ROOT=https://localhost:61798
 AUTHORITY_URI=https://www.linn.co.uk/auth/
-PROXY_ROOT=http://app.linn.co.uk
+PROXY_ROOT=https://app.linn.co.uk
 SMTP_HOSTNAME=smarthost
 SHIPFILES_FROM_ADDRESS=someone@something.co.uk
 SHIPFILE_TEMPLATE_PATH=./views/ShipfilePdfTemplate.html
-HTML_TO_PDF_API_CONVERSION_ENDPOINT=http://localhost:5000/convert/html
+HTML_TO_PDF_API_CONVERSION_ENDPOINT=https://localhost:5000/convert/html
```
Comment (lewisrenfrew) [inline src/Service.Host/config-example.env:15]: for when you update the real configs in S3
For some reason the sys healtcheck for the pdf service (on http and https) is bad gatewaying, but the real one works fine on https, e.g.
https://app.linn.co.uk/pdf-service/ping
so this shouldn't break anything

Comment (richardiphillips) [reply src/Service.Host/config-example.env:15]: i've already switched stores over in real life. Now if anyone links out to an non https friendly solution they'll have to go and fix it first!

**Pull request merged**